articles/multifactor-authentication/guardian/user-guide.md

```diff
@@ -0,0 +1,79 @@
+
+## Sign Up as a New User
+
+If you do not have an existing account, you will need to sign up to create one. Click the **SIGN UP** button and enter your email and create a password.
+
+![](/media/articles/mfa/sign-up.png)
+
+Next, you will see the option to download the Auth0 Guardian app from either the [App Store](http://www.apple.com/itunes/) or from [Google Play](https://play.google.com/store). Underneath that, is there is the option to use [Google Authenticator]() or SMS depending on the application's settings.
+ 
+![](/media/articles/mfa/choose-mfa.png)
+
+Choose the type of two-factor to use:
+
+### Guardian
+To use the Guardian app, first download the either the iPhone or Android app depending on the type of phone you have. Once you have Guardian downloaded, click **I've already downloaded it**.
+
+Next, a code will appear, you will have five minutes to scan the code before it expires. Open the Guardian app and scan the code.
+
+![](/media/articles/mfa/guardian-code.png)
+
+Once your code has been successfully scanned, you will see a confirmation screen which includes a recovery code. If for some reason you do not have your mobile device, you will need this recovery code to login. Make sure to take note of this code and keep it somewhere safe. Check the box that you have recorded the code, and then you are logged in.
+
+![](/media/articles/mfa/guardian-recovery-code.png)
+
+### Google Authenticator
+If you would prefer to use the Google Authenticator app, click on the link for Google Authenticator.  You will need to download the Google Authenticator app on your mobile device. 
+
+[Click here to learn more about Google Authenticator](https://support.google.com/accounts/answer/1066447) 
+
+A code will appear, and you will have five minutes to scan the code before it expires. After scanning the code, you will get a six digit code to enter. 
+
+![](/media/articles/mfa/google-code.png)
+
+### SMS
+
+Depending on the applications settings, SMS may be an option to use for two-factor authentication. Click on the the **SMS** link after entering your email and password.
+
+Then select your phone number's country code and enter your mobile phone number. You must be able to receive SMS to your device to use this option.
+
+![](/media/articles/mfa/sms.png)
+
+Then you should receive a six digit code in a message to your phone. Enter this code into the box. Then you will see a recovery code, make sure to note this code as you will need it to login if you do not have your device. Check the box that you have recorded the code, and then you are all set and logged in.
+
+## Logging in
+
+Depending on the type of two-factor authentication you chose when you were signing up, this will affect how you login.
+
+### Guardian
+
+After entering your username and password, you will see a scannable code appear. Scan this code using the Auth0 Guardian app. This will give you a six digit code to enter to complete your sign in.
```

Comment:
This is not right (the google authenticator below is also wrong).
The user needs to scan the code only once, and then when logging in you should receive a push notification with the actions to approve or reject the login.
In case of google authenticator, once the user scanned the code, they will need to input a code (like sms, but instead of waiting for an sms you just open the app and look at the code)